files/scripts/swss.sh

```diff
@@ -339,6 +339,23 @@ start() {
         $SONIC_DB_CLI APPL_STATE_DB FLUSHDB
         clean_up_chassis_db_tables
         rm -rf /tmp/cache
+        if [[ x"$sonic_asic_platform" == x"broadcom" ]]; then
+            . /host/machine.conf
+            if [ -n "$aboot_platform" ]; then
+                platform=$aboot_platform
+            elif [ -n "$onie_platform" ]; then
+                platform=$onie_platform
+            else
+                platform="unknown"
+            fi
+            # Need to restart PMON on media_settings.json skus due to
+            # https://github.com/sonic-net/sonic-buildimage/issues/21902
+            if [[ x"$platform" == x"x86_64-arista_7800r3a"* ]]; then
+                debug "Restarting pmon service..."
+                /bin/systemctl restart pmon
```

Comment:
@arista-nwolfe can we restart only the Xcvrd ...restarting the pmon docker seems unnecessary?

Reply:
Fixed in latest commit